test/index.js

```diff
@@ -741,6 +741,42 @@ it('accepts request object in validateFunc', (done) => {
     });
 });
 
+it('includes additional attributes in WWW-Authenticate header', (done) => {
+    const server = new Hapi.Server();
```

Comment:
:+1: 

off topic: what do you think about a separate PR to add `.eslintrc` with [hapi config](https://github.com/continuationlabs/eslint-plugin-hapi) and `.editorconfig` to match?